documentation/package.json

```diff
@@ -25,9 +25,12 @@
   },
   "dependencies": {
     "@azure/msal-browser": "2.32.2",
+    "@codemirror/lang-javascript": "^6.1.2",
     "@docusaurus/core": "2.2.0",
     "@docusaurus/preset-classic": "2.2.0",
     "@docusaurus/theme-live-codeblock": "2.2.0",
+    "@uiw/codemirror-theme-material": "^4.19.7",
+    "@uiw/react-codemirror": "^4.19.7",
```

Comment:
Pin these dependencies, such that renovate will not instantly spam us with 3 PRs doing it when this is merged.